### ISSUE TITLE
Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning.

### ISSUE THREAD

**Issue by bahador** [TehShrike/svelte-preprocess-postcss#2]
I get the following error when running rollup:
```Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning.```

here's the `svelte` config in `rollup.config.js`:
```
    svelte({
      css: function (css) {
        css.write(`public/build/styles.${hash}.css`, !production)
      },
      hydratable: true,
      preprocess: {
        style: sveltePreprocessPostcss({
          configFilePath: '',
          useConfigFile: true,
          plugins: []
        })
      }
    })
```

and here's my `postcss.config.js`:
```
module.exports = {
  from: undefined,
  plugins: []
}
```

i can fix this if i edit this line: https://github.com/TehShrike/svelte-preprocess-postcss/blob/master/index.js#L20
to be this:
`.process(css, { from: undefined })`

but this is probably wrong...

**Comment by TehShrike** [comment]
Thanks for raising this issue - I've just been ignoring that (it's just a warning, not an error).

I haven't known what, if anything, it would make sense to set the `from` to :-x

So, it's probably fine to set it to `undefined` just to make the warning go away?

**Comment by bahador** [comment]
i'm not sure, to be honest. i'd try `false` and go with that if it worked.

**Comment by bahador** [comment]
i'm not sure, to be honest. i'd try `false` and go with that if it worked.

**Comment by dalisoft** [comment]
Same issue for me

**Comment by yaliv** [comment]
Please... :hammer: it.

**Comment by TehShrike** [comment]
Has anyone tested an alternative?

**Comment by TehShrike** [comment]
Pull requests are welcome

**Comment by yaliv** [comment]
Setting `from: undefined` in `postcss.config.js` has no effect here.

And at least I've tried codes from [this PR](https://github.com/TehShrike/svelte-preprocess-postcss/pull/4) for the `from` config to work. So I think it's good to merge that one.

**Comment by TehShrike** [comment]
Published as 1.1.0 – let me know if there are any issues.
